sdks/wasm/sdk/Mono.WebAssembly.Build/Mono.WebAssembly.Build.targets

```diff
@@ -68,6 +68,16 @@
 			<_MonoWasmRuntimePath Condition="'$(_MonoWasmDebugRuntime)'=='true'">$(MonoWasmRuntimePath)threads-debug\</_MonoWasmRuntimePath>
 		</PropertyGroup>
 
+		<PropertyGroup Condition="'$(EnableMonoWasmDynamic)'=='true' And Exists('$(MonoWasmRuntimePath)builds\')">
+			<_MonoWasmRuntimePath>$(MonoWasmRuntimePath)builds\dynamic-release\</_MonoWasmRuntimePath>
+			<_MonoWasmRuntimePath Condition="'$(_MonoWasmDebugRuntime)'=='true'">$(MonoWasmRuntimePath)builds\dynamic-debug\</_MonoWasmRuntimePath>
+		</PropertyGroup>
+
+		<PropertyGroup Condition="'$(EnableMonoWasmDynamic)'=='true' And !Exists('$(MonoWasmRuntimePath)builds\')">
```

Comment:
I'm not too sure of the reason of having two groups, one with `build/` and another without. Should I keep the second one ?

Reply:
it's subtle but one is `a && b` and the other is `a && !b` but the decision here to just ignore the missing directory is very strange and magical, i don't like it. i in fact was told to look in builds/ in the past and it wasn't there because of this backup target... we should consider whether we can remove it and just make the target autocreate the directory

Reply:
Maybe @kjpou1 knows more ?

Reply:
It looks like it may be used for two different scenarios, one is for the SDK build itself, the other one may be for an app build.

Reply:
This is to support `<MonoWasmSdkPath></MonoWasmSdkPath>` build property.  One can set the path here and it will use the local build from this path instead of the packages.